### ISSUE TITLE
theory name in imports is broken

### ISSUE THREAD

**Issue by m-iancu** [KWARC/sTeX#132]
After the update it is still broken, now it's just an extra slash (I guess escaping failed) : 
`<omdoc:imports from="smglom/mv/source/equal\#2 ..."`

**Comment by kohlhase** [comment]
The problem is in ```smultiling.sty.ltxml```, in the definition of the ```modnl``` environment. I have tried changing the string interpolation, could you please try again? 

I am assigning this to @angerhang he knows this better.

**Comment by m-iancu** [comment]
Latest version is back to the "no slash" version: 
```
<omdoc:imports from="smglom/mv/source/equal#2"
  xml:id="equal.en.imports2" about="#equal.en.imports2"
  stex:srcref="smglom/mv/source/equal.en.tex#textrange(from=1;0,to=6;11)"
/>

```

**Comment by kohlhase** [comment]
hmmmm, somehow my changes semm to have been reverted. I made the changes again, could you please try again? (sorry, for me nothing works).

**Comment by m-iancu** [comment]
Same as above, not sure if commit went through (https://github.com/KWARC/sTeX/commits/master)

**Comment by kohlhase** [comment]
@dginev, could you have a look at this as well. Another blocker on MathHub

**Comment by dginev** [comment]
Here is the culprit: https://github.com/KWARC/sTeX/commit/9714311507a710b186d885ec2a803331391ff7b9#commitcomment-13450514

**Comment by angerhang** [comment]
So what exactly do we want here?

I understand we don't want the extra slash in the import path, but what should it be instead?

**Comment by kohlhase** [comment]
we want 
```
After the update it is still broken, now it's just an extra slash (I guess escaping failed) :
<omdoc:imports from="smglom/mv/source/equal\#2 ..."
```
where instead of ```\#2``` we have the theory name it was called with, i.e. something like ```equal/mv```.

**Comment by angerhang** [comment]
This is fixed.

**Comment by kohlhase** [comment]
Actually, this is not fixed completely yet. Converting ```MiKoMH/GenCS/source/dmath/en/functions.tex``` I get 
```
    <omdoc:imports from="../../../../Users/kohlhase/localmh/MathHub/MiKoMH/GenCS/source/dmath/en/relation-operations#2" xml:id="myfunctions.imports1" about="#myfunctions.imports1" stex:srcref="MiKoMH/GenCS/source/dmath/en/functions.tex#textrange(from=2;0,to=2;71)"/>
    <omdoc:imports conservative="false" from="../../../../Users/kohlhase/localmh/MathHub/smglom/numberfields/source/arithmetics#2" xml:id="myfunctions.imports2" about="#myfunctions.imports2" stex:srcref="MiKoMH/GenCS/source/dmath/en/functions.tex#textrange(from=3;0,to=3;42)"/>
``` 
from
```
\importmhmodule[path=dmath/en/relation-operations]{relation-operations}
\gimport[smglom/numberfields]{arithmetics}
```
This would be very important to fix.

**Comment by angerhang** [comment]
I think the changes I made in this https://github.com/KWARC/sTeX/pull/137/files pull request disappeared somehow.
The `<omdoc:import>` is back to the not working version.
```perl
   '<omdoc:imports from="?&GetKeyVal(#1,"load")(&canonical_omdoc_path(&GetKeyVal(#1,"load")))()\##2">'
```
See the current version at: https://github.com/KWARC/sTeX/blob/master/sty/modules/modules.sty.ltxml
Don't know exactly why, but it might because I deleted my fork after the merge request. However, deleting a fork after the pull request is accepted shouldn't cause something like this though.

I will make the same changes again but on a branch this time.
